src/scenes/PlantingSitesRouter/edit/PlantingSiteSelectTypeModal.tsx

```diff
@@ -39,7 +39,7 @@ export default function PlantingSiteSelectTypeModal(props: PlantingSiteSelectTyp
     <DialogBox
       scrolled
       onClose={handleClose}
-      open={open}
+      open={true}
```

Comment:
Could also omit the `={true}` here

```suggestion
      open
```